package.json

```diff
@@ -794,6 +794,12 @@
             "default": true,
             "markdownDescription": "Enable/disable inlay hints for namespaces.",
             "scope": "resource"
+          },
+          "robotcode.testView.enabled": {
+            "type": "boolean",
+            "default": true,
+            "description": "Whether to show robot tests in the test view. You may want to disable this if you are using another test runner (eg. https://github.com/DetachHead/pytest-robotframework)",
```

Comment:
1. Again, the view name is "Test Explorer" ...
 
2. I know you don't want to advertise your extension, but at the moment I don't want external links in the RobotCode settings unless the link leads to RobotFramework or RobotCode documentation. This is not against you, but I am responsible for security issues and your link is not under my control.

Reply:
yeah fair enough. i would be annoyed if someone made a PR to my project with an ad for their own lol. i wasn't intending for it to be an advertisement but more of an explanation as to why the option exists. but yeah i see how it can be interpreted as an ad so i'll remove it